frontend/.gitignore

```diff
@@ -4,6 +4,7 @@ dist-ssr
 
 # Auto generated
 src/bindings/*
+src/generated/*
```

Comment:
Why not directly generated? Or simply doc?

Reply:
Because I thought that there may be a docs folder in the future where the manually written `.mdx` files to be included in the Storybook will be.

They have to live in the `src` folder for Storybook to pick them up.

Reply:
Please shortly describe what these manually written mdx files would be for.

Reply:
@markus2330 Like this 8e9f3aaeac0045bf9f01b33db7e7a80bfa66847b?